yellowbrick/draw.py

```diff
@@ -89,3 +90,74 @@ def manual_legend(g, labels, colors, **legend_kwargs):
     # Return the Legend artist
     return g.legend(handles=handles, **legend_kwargs)
 
+
+def bar_stack(data, ax=None, labels=None, ticks=None, colors=None, 
+              orientation='vertical', colormap=None, **kwargs):
+    """
+    An advanced bar chart plotting utility that can draw bar and stacked bar charts from
+    data, wrapping calls to the specified matplotlib.Axes object. 
+
+    Parameters
+    ----------
+    data : 2D array-like 
+        The data associated with the bar chart where the columns represent each bar
+        and the rows represent each stack in the bar chart. A single bar chart would 
+        be a 2D array with only one row, a bar chart with three stacks per bar would 
+        have a shape of (3, b).  
```

Comment:
We might need to make this a bit more readable and understandable. @DistrictDataLabs/team-oz-maintainers does this make sense to you? Any thoughts?

Reply:
I picked it up from the #525. It had a basic function signature. We can change this if you suggest.